src/ToolboxBundle/DependencyInjection/Configuration.php

```diff
@@ -317,6 +321,7 @@ public function getConfigNode(NodeDefinition $rootNode)
                                             ->children()
                                                 ->scalarNode('tag')->end()
                                                 ->scalarNode('class')->end()
+                                                ->scalarNode('attr')->end()
```

Comment:
We only allow attributes without values, right? Should we validate the value? (like `id="df"` should not be valid).

Reply:
In some cases it make sense to add a value (uk-grid="masonry: true").